### PR TITLE
Lint: new lint command, lint sources, add missed docstrings, suppress issues for future

### PR DESCRIPTION
**Pull request by pkulev**
**Closes issue(s):** [ # ]

**Description of the changes:**

  * What's new: pytest-pylint module, update **make lint** target
  * Fixes: make line counting more pretty
  * Refactoring: lint all major issues, mark and suppress issure I can't or won't wix now

### CODE REVIEW

File: xoinvader/tests/test_collision.py [hunk 1]
```diff
@@ -73,19 +76,23 @@ def test_collision_manager():
...
+        def _add_collision(self, other_type, callback=None):
+            pass
+
     m = MockManager()
     Collider.__manager__ = m
     c = Collider("", [], None)
     mm = CollisionManager()
     with pytest.raises(ValueError):
-        mm._add_collision(c, "other", lambda: _)
+        mm._add_collision(c, "other", lambda: 0)
```
Comment (taptap) [inline xoinvader/tests/test_collision.py]: Why?
`lambda: _` is a good and understandable pattern, unlike `lambda: 0`. I'm against this change.

File: xoinvader/ship.py [hunk 2]
```diff
@@ -289,23 +300,24 @@ def __init__(self, pos, border):
         super(Playership, self).__init__(pos, border)
 
         self._image = Surface([
-            [' ', ' ', 'O', ' ', ' '],
-            ['<', '=', 'H', '=', '>'],
-            [' ', '*', ' ', '*', ' ']])
+            "  O  ",
+            "<=H=>",
+            " * * ",
```
Comment (taptap) [inline xoinvader/ship.py]: Much better!

File: xoinvader/tests/test_collision.py (hunk 1 above)
Comment (pkulev) [inline xoinvader/tests/test_collision.py]: _ is valid variable name that not declared in this scope.

Comment (taptap) [inline xoinvader/tests/test_collision.py]: Oops, sorry. Indeed.

**Pull request merged**